packages/flutter/test/cupertino/picker_test.dart

```diff
@@ -240,11 +240,6 @@ void main() {
         1000.0,
       );
 
-      expect(
```

Comment:
Then assert that something is on screen to indicate that it went past the edge.

Reply:
Added.